crates/bevy_reflect/src/impls/std.rs

```diff
@@ -217,6 +218,133 @@ impl<T: FromReflect> FromReflect for Vec<T> {
     }
 }
 
+impl<T: FromReflect> Array for VecDeque<T> {
```

Comment:
It seems like this implementation is also exactly the same as the `Vec<T>` impl (minus the Vec/VecDeque type haha).
Since they're basically the exact same, imo, it's worth making a macro therefore we can DRY the code up a bit.
e.g.
```
macro_rules! vec_like_reflection { .. }

vec_like_reflection!(Vec);
vec_like_reflection!(VecDeque);
```

Reply:
You'd probably need to specify the `push` method to be used as well